src/replays/_replayswidget.py

```diff
@@ -322,7 +347,9 @@ def updatemyTree(self):
 
                     # Parse replayinfo into data
                     if item.info.get('complete', False):
-                        t = time.localtime(item.info.get('launched_at', item.info.get('game_time', time.time())))
+                        # 'game_time' is 'launched_at' for older replays (if that fails maybe 'game_end' is there)
```

Comment:
If we're tinkering with the 't' and folders already, maybe we can put replays without that info into a separate bucket?